web/elm/src/SideBar/Styles.elm

```diff
@@ -141,43 +141,55 @@ opacityAttr opacity =
                 "1"
 
 
-teamIcon : Opacity -> Html.Html msg
-teamIcon opacity =
+colorAttr : Opacity -> Html.Attribute msg
+colorAttr opacity =
+    style "color" <|
+        case opacity of
+            Dim ->
+                Colors.sideBarTextDim
+
+            GreyedOut ->
+                Colors.sideBarTextDim
```

Comment:
Should this be the same as `Dim`? And if so, should we get rid of either `Dim` or `GreyedOut`, since they now function identically?

Reply:
Also, should we get rid of `opacityAttr` altogether, and rename the `Opacity` custom type to something more appropriate?

Reply:
i just realized the pin menu uses this, let me rethink this